e2e-tests/cypress/support/appeal-submission-site-health-and-safety-issues/confirmHealthAndSafetyPage.js

```diff
@@ -1,4 +1,4 @@
 module.exports = () => {
-  cy.url().should('contain', '/appellant-submission/site-access-safety');
+  cy.url().should('contain', '/appeal-householder-decision/site-access-safety');
```

Comment:
please can the dir here be updated to reflect `appeal-householder-decision` - thanks